manifests/postgres-operator.yaml

```diff
@@ -269,7 +269,7 @@ spec:
       serviceAccountName: postgres-operator
       containers:
         - name: postgres-operator
-          image: docker.io/flanksource/postgres-operator:{{.postgresOperator.version}}
+          image: registry.opensource.zalan.do/acid/postgres-operator:{{.postgresOperator.version}}
```

Comment:
I think we need to mirror this to flanksouree dockerhub as I don't think registry.opensource.zalan.do is a production-grade registry

Reply:
Can do - would you like me to mirror a range of releases, or just 1.6.2 ?